build/lib/test/test.js

```diff
@@ -221,7 +221,7 @@ async function addTiAppProperties() {
 	const tiapp_xml_string = await fs.readFile(tiapp_xml, 'utf8');
 	const content = [];
 	const insertManifest = () => {
-		content.push('\t\t\t<application android:theme="@style/Theme.Titanium.Dark">');
+		content.push('\t\t\t<application android:theme="@style/Theme.Titanium.Dark"  android:icon="@mipmap/ic_launcher">');
```

Comment:
@m1ga I think the default icon which SDK puts in is `android:icon="@drawable/appicon"`. I do not see any `mipmap` folder in a fresh project.

Reply:
we have mipmaps since 12.4.0: https://github.com/tidev/titanium-sdk/tree/098f8c2785ef3257b61736141d63ffd61b404ef8/templates/app/default/template/platform/android/res

a new project looks like this:
![Bildschirmfoto_20250128_103355](https://github.com/user-attachments/assets/2e798d0f-7bc4-48a3-bdea-b11836d4d02b)

Reply:
Yeah, I just double checked, I think I changed the platform folder during some tests. It's fine then!